src/lib/hooks/usePaymentTokenInfo.ts

```diff
@@ -5,14 +5,19 @@ import { useZnsSdk } from './useZnsSdk';
 export const usePaymentTokenInfo = (token: string) => {
 	const sdk = useZnsSdk();
 
-	return useQuery(
-		['domain-payment-token-info', token],
-		async () => await sdk.zauction.getPaymentTokenInfo(token),
+	const query = useQuery(
+		['token', 'info', { token }],
+		() => sdk.zauction.getPaymentTokenInfo(token),
 		{
 			retry: false,
 			refetchOnMount: false,
 			refetchOnWindowFocus: false,
 			enabled: Boolean(token),
 		},
 	);
+
+	return {
+		...query,
+		isLoading: query.isLoading || query.isIdle,
```

Comment:
Nice, I can use this in Place Bid rather than fetching data in a higher component